src/main/flight/mixer.c

```diff
@@ -766,17 +800,59 @@ float applyThrottleLimit(float throttle) {
     return throttle;
 }
 
-FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs) {
+FAST_RAM_ZERO_INIT float lastRcDeflection[3], stickMovement[3];
+
+void applyAirMode(float *motorMix, float motorMixMax) {
+    float motorMixNormalizationFactor = motorMixRange > 1.0f && hardwareMotorType != MOTOR_BRUSHED ? motorMixRange : 1.0f;
+    float motorMixDelta = 0.5f * motorMixRange;
+    float authorityZeroThrottle, authorityFullThrottle;
+
+    if (isAirmodeActive()) {
+        authorityZeroThrottle = authorityFullThrottle = 1.0f;
+    } else {
+        float authorityMultiplier = predictiveAirModeAuthorityMultiplier ? calculatePredictiveAirModeAuthorityMultiplier() : 1.0f;
+        authorityZeroThrottle = MAX(minAuthorityZeroThrottle * authorityMultiplier, 1.0f);
+        authorityFullThrottle = MAX(minAuthorityFullThrottle * authorityMultiplier, 1.0f);
```

Comment:
I am not sure if this makes sense to me. This will always result into 1? minAuthorityZeroThrottle * authorityMultiplier cannot exceed 1 in  calculatePredictiveAirModeAuthorityMultiplier()

Reply:
Authority multiplier can be greater than 1. In fact authority multiplier can only be 1 or greater than 1.

Reply:
Yes, I have to fix it, it's an error I made refactoring the Predictive AirMode logic. Thanks you!